include/aws/http/http2_stream_manager.h

```diff
@@ -97,6 +97,26 @@ struct aws_http2_stream_manager_options {
     void *shutdown_complete_user_data;
     aws_http2_stream_manager_shutdown_complete_fn *shutdown_complete_callback;
 
+    /**
+     * Optional.
+     * When set, connection will be closed if 5xx response received from server.
+     */
+    bool close_connection_on_server_error;
+    /**
+     * Optional.
+     * The period for all the connections held by stream manager to send a PING in seconds.
+     * If you specify 0, manager will NOT send any PING.
+     * Note: if set, it must be large than the time of ping timeout setting.
+     */
+    size_t connection_ping_period_sec;
+    /**
+     * Optional.
+     * Network connection will be closed if a ping response is not received
+     * within this amount of time (nanoseconds).
+     * If you specify 0, a default value of 3 seconds is used.
+     */
+    size_t connection_ping_timeout_ns;
```

Comment:
using sec for one and ns for the other feels weird to me. How about we just make both of them ms? seems like a happy medium

Reply:
Hmmm, I assume user will want to have small timeout time setting and a much longer ping period setting. 

For MQTT, we also have `keep_alive_time_secs` and `ping_timeout_ms`. Maybe it's better to have `connection_ping_timeout_ms` instead though, no one wants to use ns I guess.

Reply:
I still hate that these use two different units of time
Please use `ms` for both
MQTT uses `sec` because that's required by the spec